ntbs-integration-tests/NotificationPages/PatientPageTests.cs

```diff
@@ -132,6 +136,7 @@ public async Task Post_RedirectsToNextPageAndSavesContent_IfModelValid()
             Assert.Equal(EthnicityId, ((IHtmlSelectElement)reloadedDocument.GetElementById("Patient_EthnicityId")).SelectedIndex.ToString());
             Assert.True(((IHtmlInputElement)reloadedDocument.GetElementById("sexId-2")).IsChecked);
             Assert.Equal(CountryId, ((IHtmlSelectElement)reloadedDocument.GetElementById("Patient_CountryId")).SelectedIndex.ToString());
+            Assert.Equal(LocalPatientId, ((IHtmlInputElement)reloadedDocument.GetElementById("Patient_LocalPatientId")).Value);

```

Comment:
Definitely fine to check in without addressing this!
We could probably add another extension for cleaning the assert blocks on this test page.

Reply:
Completely agree, with both parts of that comment. I'll merge this piece in now to keep it moving along, but definitely an improvement worth introducing soon!